org.sugarlabs.ReadETexts.json

```diff
@@ -240,8 +211,8 @@
                 {
                     "type": "git",
                     "url": "https://github.com/espeak-ng/espeak-ng.git",
-                    "tag": "1.49.2",
-                    "commit": "3ed34d3a74eb69cedcdfa98dbedef4359279d352"
+                    "tag": "1.50",
```

Comment:
are you sure this version is compatible?

Reply:
I thought I need to get the latest version, maybe thats the reason, the activity doesn't start